### ISSUE TITLE
Reloading the same page shouldn't show cached version of the page

### ISSUE THREAD

**Issue by artm** [turbolinks/turbolinks#480]
```js
Turbolinks.visit(window.location, { action: "replace" });
```

Replaces **current version** of the page with the cached **previous version** of the page before replacing it with the **next version** of the page. It shouldn't.

**Comment by agorf** [comment]
Happens to me as well. Just to clarify: it does not happen on reload but on link re-click.

**Comment by agorf** [comment]
Btw adding the following in the `<head>` of the document solved it for me:

```html
<meta name="turbolinks-cache-control" content="no-preview">
```

More here: https://github.com/turbolinks/turbolinks#opting-out-of-caching

**Comment by artm** [comment]
yep, we're doing that as well, but this means no preview when coming from a different page either.

**Comment by tleish** [comment]
I believe you could add one more line of code to avoid preview when navigating to same page, but all preview when coming from a different page.

```javascript
document.head.insertAdjacentHTML('beforeend', `<meta name="turbolinks-cache-control" content="no-preview">`);
Turbolinks.visit(window.location, { action: "replace" });
```
